client/src/pages/SearchSitter/SearchSitter.tsx

```diff
@@ -0,0 +1,89 @@
+import { useEffect, FormEvent, useState, ChangeEvent } from 'react';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import SearchIcon from '@material-ui/icons/Search';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import TextField from '@material-ui/core/TextField';
+import Typography from '@material-ui/core/Typography';
+import useStyles from './useStyles';
+import SitterCard from '../../components/SitterCard/SitterCard';
+import { Profile } from '../../interface/User';
+import { Button, InputAdornment } from '@material-ui/core';
+import { getAllSitterProfiles } from '../../helpers/APICalls/getSitterProfile';
+import { searchProfiles } from '../../helpers/APICalls/searchUsers';
+
+export default function SearchSitter(): JSX.Element {
+  const classes = useStyles();
+  const [search, setSearch] = useState<string>('test');
+  const [allSitters, setAllSitters] = useState<Profile[]>([]);
+
+  useEffect(() => {
+    getAllSitters();
+  }, []);
```

Comment:
Make sure you understand the implications of using this dependencies array. Leaving it empty means your `effect` runs only once on mount and never re-runs.